include/ll.h

```diff
@@ -0,0 +1,29 @@
+#ifndef __BM_LL_H__
+#define __BM_LL_H__
```

Comment:
I wrote up an item in our team notes that we should discuss in our next meeting and agree an a standard for include guards / pragma once / neither until needed.